src/scales.js

```diff
@@ -413,6 +413,8 @@ function exposeScale({
     // band, point
     ...scale.align && {align: scale.align(), round: scale.round()},
     ...scale.padding && (scale.paddingInner ? {paddingInner: scale.paddingInner(), paddingOuter: scale.paddingOuter()} : {padding: scale.padding()}),
+    ...type === "point" && {step: scale.step()},
+    ...type === "band" && {bandwidth: scale.bandwidth(), step: scale.step()},
```

Comment:
```suggestion
    ...scale.bandwidth && {bandwidth: scale.bandwidth(), step: scale.step()},
```

I think it’s fine (and possibly helpful) to return a bandwidth of zero for point scales and to reduce the variation in response shapes.